templates/single-course/modules.php

```diff
@@ -106,7 +106,7 @@
 	                                    $course_id = Sensei()->lesson->get_course_id( get_the_ID() );
 	                                    if ( Sensei_Utils::is_preview_lesson( get_the_ID() ) && ! Sensei_Utils::user_started_course( $course_id, get_current_user_id() )  ) { ?>
 
-	                                        <span class="preview-label"><?php _e( 'Free Preview', 'woothemes-sensei' ); ?></span>
+	                                        <?php echo Sensei()->frontend->sensei_lesson_preview_title_tag( $course_id ); ?>
```

Comment:
Might as well put this inside of the previous `<?php ?>` instead of creating another one.

Reply:
Done!